bedrock/mozorg/templates/mozorg/home/includes/survey-b.html

```diff
@@ -0,0 +1,6 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% stylesheet 'home-survey' %}
```

Comment:
I now realize the other homepage survey is already doing this too. Up to you if you want to address this or not (it's more of a query as I don't see any real issues).